### PR TITLE
Create SECURITY.md

### LINKED ISSUE #200: Feature: Security Policy
Please add a `SECURITY.md` as to define authd's Security Policy.

The primary purpose of a Security Policy is to tell researchers how to privately report issues to authd. `SECURITY.md` is the standard place to write the policy and will be displayed on https://github.com/ubuntu/authd/security 

It's great that the Issue template also includes a reporting policy! That text asks that vulnerabilities should be reported on Launchpad. Please specify that security issues should be reported on Launchpad using the "This bug is a security vulnerability" option, otherwise the bug will not be private.

[LXD](https://github.com/canonical/lxd/security) and [GitHub](https://securitylab.github.com/advisories/#policy) have great examples. Happy to review.

### PR DESCRIPTION
**Pull request by jibel**
Added SECURITY.md

closes #200 
UDENG-2243

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+apt policy authd
+```
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability within this repository, we encourage responsible disclosure. Please report any security issues to help us keep `authd` secure for everyone.
+
+### Private Vulnerability Reporting
+
+The most straightforward way to report a security vulnerability is via [GitHub](https://github.com/canonical/lxd/security/advisories/new).
```
Comment (didrocks) [inline SECURITY.md:20]: This is the lxd repo, we know where you copied from! :p

**Review by didrocks — APPROVED** [review]
One url to fix and then, you can merge!

File: SECURITY.md (hunk 1 above)
Comment (jibel) [reply SECURITY.md:20]: ah,  I've read it ten times :D

Comment (didrocks) [reply SECURITY.md:20]: I’m unsure the 11th times would have fixed it TBH :p Always better with fresh eyes!

**Review by didrocks — APPROVED** [review]

**Pull request merged**